packages/web-twig/tests/components-fixtures/buttonWithMainProps.twig

```diff
@@ -1,7 +1,7 @@
 {% set onClickCallback = "alert('\" test')" %}
 
-<Button id="testId" name="testName" data-main="main" aria-label="label" not-valid-prop="unexist" color="primary" onclick={onClickCallback}>button</Button>
-<Button id="testId" name="testName" data-main="main" aria-label="label" not-valid-prop="unexist" color="primary" onclick={onClickCallback}>button</Button>
+<Button id="testId" name="testName" data-main="main" aria-label="label" not-valid-prop="unexist" color="primary" onclick={onClickCallback} formtarget="_blank">button</Button>
+<Button id="testId" name="testName" data-main="main" aria-label="label" not-valid-prop="unexist" color="primary" onclick={onClickCallback} formtarget="_blank">button</Button>
```

Comment:
Because of [the testing `onClick` vs. `onclick` attributes](https://github.com/lmc-eu/spirit-design-system/commit/ab62f78f5ce07eda6c561b89901b4ac01f2704dd#diff-0ba83c2ecd2cdb345eb51e4cf895775dcfb15cc83449042471e0be8509eff831). I'll remove the duplicated row.